src/idiomatic/foundations-api-design/meaningful-doc-comments/avoid-redundancy.md

```diff
@@ -0,0 +1,55 @@
+---
+minutes: 15
+---
+
+# Avoiding Redundancy
+
+Function names and type signatures already document some information, avoid
+repeating them!
+
+```rust
+// Don't do this!
+/// Parses an ipv4 from a str. Returns an option for failure modes.
+fn parse_ip_addr_v4(input: &str) -> Option<IpAddrV4> { ... }
+
+// TODO: couple more of these, for the instructor to go through with students.
+```
+
+<details>
+- Motivation: Documentation that repeats name/signature information provides nothing new to the API user.
```

Comment:
Second motivation: Documentation that repeats API signatures can go out of date, while the API will always stay as the source of truth.